### DISCUSSION TITLE
How to use kafka-ui in an environment where public network is not possible

### DISCUSSION THREAD

**Discussion by christopher-crispy** [provectus/kafka-ui#3942] [Q&A]
The network environment cannot be connected to the outside, and proxy cannot be established.
How do I use kafka-ui in this environment?
When the web screen opens for the first time, I can't connect because I get various packets from the public.

**Comment by Haarolean** [comment]
Which version are you on?
Please elaborate on "various packets from the public". Provide a list of failed URLs.
[This](https://github.com/provectus/kafka-ui/issues/3865) is the last known issue with air gap envs. The rest should work.

**Comment by waitspring** [comment]
When the network environment cannot be connected to the outside, maybe you need login the `Kafka-UI` host, and try:

```bash
curl -I http://ip:port
```

If your command return `200` code so that your `Kafka-UI` service is ok, you need check your network environment, else, your `Kafka-UI` service is error.
